### PR TITLE
password length validator for standalone registration component

### PR DESCRIPTION
**Pull request by crunck78**

### CODE REVIEW

**Comment by crunck78** [comment]
@AnthonyNahas yes, I will do so.

**Comment by crunck78** [comment]
I have updated the Readme for the target component under the api.

File: projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts [hunk 1]
```diff
@@ -71,6 +73,9 @@ export class NgxAuthFirebaseuiRegisterComponent implements OnInit, OnDestroy {
   @Input() passwordConfirmationText = 'Password Confirmation';
   @Input() passwordConfirmationErrorRequiredText = 'Password confirmation is required';
   @Input() passwordErrorMatchText = 'Password must match';
+  // crunck78 - request feature password length control for standalone registration component
```
Comment (geromegrignon) [inline projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts:76]: i would suggest to remove the comment as there is no added value.

File: projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts [hunk 2]
```diff
@@ -116,7 +125,9 @@ export class NgxAuthFirebaseuiRegisterComponent implements OnInit, OnDestroy {
     this.registerForm = this.formBuilder.group({
       name: ['', Validators.required],
       email: ['', [Validators.required, Validators.email]],
-      password: ['', Validators.required],
+      password: ['', [Validators.required,
+                     Validators.minLength(this.config.passwordMinLength), 
+                     Validators.maxLength(this.config.passwordMaxLength)]], //TODO add Validator password
```
Comment (geromegrignon) [inline projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts:130]: i would remove the TODO as the scope of the PR is the length validator.

Comment (crunck78) [reply projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts]: Removed. Thanks.

File: projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts (hunk 1 above)
Comment (crunck78) [reply projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts]: Removed.

**Comment by crunck78** [comment]
Much appreciated.  Thanks.

**Pull request merged**